devfile.yaml

```diff
@@ -0,0 +1,54 @@
+schemaVersion: 2.1.0
+metadata:
+  name: nodejs-mongodb
+components:
+  - name: nodejs
+    container:
+      image: quay.io/eclipse/che-nodejs10-ubi:next
+      env:
+      # The values below are used to set up the environment for running the application
+        - name: SECRET
+          value: 220fd770-c028-480d-8f95-f84353c7d55a 
+        - name: NODE_ENV
+          value: production
+      endpoints:
+        - exposure: public
+          name: nodejs
+          targetPort: 8080
+      memoryLimit: 1G
+      mountSources: true
+  - name: mongo
+    container:
+      image: quay.io/eclipse/che--centos--mongodb-36-centos7:latest-a915db7beca87198fcd7860086989fe8a327a1a4f6508025b64ab28fcc7423b2
```

Comment:
not directly related to this PR but probably here we would need to move to a more recent image as well

mongodb is at version 5.0 now